README.md

```
@@ -126,33 +73,18 @@ devlovers.net/
-Task tracking via [GitHub Projects](https://github.com/DevLoversTeam/devlovers.net/projects) board.
+- Personal learning dashboard with progress insights
+- Quiz history and performance overview
+- Quick access to key account activity
 
-## License
+![Dashboard](./assets/09-screencapture.png)
 
-**MIT**
+> [!IMPORTANT]
+> **Contact me:** [contact@devlovers.net](mailto:contact@devlovers.net)
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)